templates/registration/login.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  {% if form.errors %}
+    <p style="color: red">Invalid credentials</p>
+  {% endif %}
+  <form action="{% url 'login'%}" method="post">

```

Comment:
Consider adding a space before the closing `%}` in the URL tag for consistency and readability. Change `{% url 'login'%}` to `{% url 'login' %}`.